src/types/User.ts

```diff
@@ -1,4 +1,19 @@
-export interface Experience {
+export enum PlaceInTalkShowStudioVenue {
+  stage = "stage",
+  audience = "audience",
+  requesting = "requesting",
+}
+
+export interface TalkShowStudioExperience {
+  place?: PlaceInTalkShowStudioVenue;
+  isSharingScreen?: boolean;
+  isMuted?: boolean;
+  isUserCameraOff?: boolean;
+  cameraClientUid?: string;
+  screenClientUid?: string;
+}
```

Comment:
This is something I want to come back to and review/think about more in future.. not sure if this is a good location for it/whether we need all of these things here.